### PR TITLE
Additional logging for polkadot network protocols

### PR DESCRIPTION
**Pull request by arkpar**

### CODE REVIEW

File: node/network/bitfield-distribution/src/lib.rs [hunk 1]
```diff
@@ -485,24 +496,51 @@ where
...
+				?view,
+				"Peer view change",
+			);
 			handle_peer_view_change(ctx, state, peerid, view).await;
 		}
 		NetworkBridgeEvent::OurViewChange(view) => {
+			tracing::trace!(
+				target: LOG_TARGET,
+				?view,
+				"Peer view change",
```
Comment (rphmeier) [inline node/network/bitfield-distribution/src/lib.rs:531]:
```suggestion
				"Our view change",
```

**Review by rphmeier — APPROVED** [review]

**Comment by rphmeier** [comment]
bot merge

File: node/network/availability-distribution/src/responder.rs [hunk 2]
```diff
@@ -99,5 +105,14 @@ where
...
 	.await;
 
-	rx.await.map_err(|e| Error::QueryChunkResponseChannel(e))
+	rx.await.map_err(|e| {
+		tracing::trace!(
+			target: LOG_TARGET,
+			?validator_index,
+			?candidate_hash,
+			error = ?e,
+			"Error retrieveing chunk",
```
Comment (rphmeier) [inline node/network/availability-distribution/src/responder.rs:114]:
```suggestion
			"Error retrieving chunk",
```

**Comment by rphmeier** [comment]
bot no

**Comment by rphmeier** [comment]
bot cancel

**Comment by rphmeier** [comment]
bot merge

**Comment by rphmeier** [comment]
bot merge

**Comment by arkpar** [comment]
bot merge

**Pull request merged**